### ISSUE TITLE
[Bug]:  cgroup-name.sh cannot find the name of docker container

### ISSUE THREAD

**Issue by instantdreams** [netdata/netdata#12070]
### Bug description

I am running netdata on two Raspberry Pis running latest Raspberry Pi OS Lite, and one Intel NUC running latest Debian.

I get the following message:
```
$ docker logs netdata 2>&1 | grep docker
2022-02-02 04:27:49: cgroup-name.sh: WARNING: cannot find the name of docker container '[container id]'
```

The containers in netdata web ui (and therefore Prometheus and Grafana) show IDs rather than names.

### Expected behavior

I have followed the steps to give [group access to the docker socket](https://learn.netdata.cloud/docs/agent/packaging/docker/#giving-group-access-to-the-docker-socket-less-safe).

I expected the containers to display names rather than IDs.

### Steps to reproduce

I am running netdata on two Raspberry Pis running latest Raspberry Pi OS Lite, and one Intel NUC running latest Debian.

When checking the logs I get the following message:
```
$ docker logs netdata 2>&1 | grep docker
2022-02-02 04:27:49: cgroup-name.sh: WARNING: cannot find the name of docker container '[container id]'
```

I have confirmed the following for the Pis:
```
$ grep docker /etc/group | cut -d ':' -f 3
995

$ docker exec netdata env
PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
HOSTNAME=[hostname]
PGID=995
TZ=[time zone]
NETDATA_OFFICIAL_IMAGE=true
DOCKER_GRP=netdata
DOCKER_USR=netdata
NETDATA_LISTENER_PORT=19999
HOME=/root

$ docker exec netdata id netdata
uid=201(netdata) gid=201(netdata) groups=201(netdata),201(netdata),995(docker)

$ docker-compose config
services:
  netdata:
    cap_add:
    - SYS_PTRACE
    container_name: netdata
    environment:
      PGID: '995'
      TZ: [timezone]
    hostname: [hostname]
    image: netdata/netdata
    ports:
    - published: 19999
      target: 19999
    restart: unless-stopped
    security_opt:
    - apparmor:unconfined
    volumes:
    - /srv/netdata/config:/etc/netdata:rw
    - netdatalib:/var/lib/netdata:rw
    - netdatacache:/var/cache/netdata:rw
    - /etc/passwd:/host/etc/passwd:ro
    - /etc/group:/host/etc/group:ro
    - /proc:/host/proc:ro
    - /sys:/host/sys:ro
    - /etc/os-release:/host/etc/os-release:ro
version: '3'
volumes:
  netdatacache: {}
  netdatalib: {}
```

I have the same results for the NUC except the group ID is 998 and this is set accordingly:
```
$ grep docker /etc/group | cut -d ':' -f 3
998

$ docker exec netdata env
[...]
PGID=998
[...]

$ docker exec netdata id netdata
uid=201(netdata) gid=201(netdata) groups=201(netdata),201(netdata),998(docker)

$  docker-compose config
services:
  netdata:
    cap_add:
    - SYS_PTRACE
    container_name: netdata
    environment:
      PGID: '998'
      TZ: [timezone]
    hostname: [hostname]
    image: netdata/netdata
    ports:
    - published: 19999
      target: 19999
    restart: unless-stopped
    security_opt:
    - apparmor:unconfined
    volumes:
    - /srv/netdata/config:/etc/netdata:rw
    - netdatalib:/var/lib/netdata:rw
    - netdatacache:/var/cache/netdata:rw
    - /etc/passwd:/host/etc/passwd:ro
    - /etc/group:/host/etc/group:ro
    - /proc:/host/proc:ro
    - /sys:/host/sys:ro
    - /etc/os-release:/host/etc/os-release:ro
version: '3'
volumes:
  netdatacache: {}
  netdatalib: {}
```

Additionally the docker images were refreshed to ensure they are the latest image (but were installed fresh yesterday):
```
$ docker ps --all --format "table {{.Names}}\t{{.Size}}\t{{.Status}}\t{{.RunningFor}}\t{{.State}}"
NAMES                   SIZE                      STATUS                    CREATED          STATE
netdata                 3.56kB (virtual 393MB)    Up 22 minutes (healthy)   22 minutes ago   running
```

### Installation method

docker

### System info

```shell
$ uname -a; grep -HvE "^#|URL" /etc/*release
Linux [servername] 5.10.0-10-amd64 #1 SMP Debian 5.10.84-1 (2021-12-08) x86_64 GNU/Linux
/etc/os-release:PRETTY_NAME="Debian GNU/Linux 11 (bullseye)"
/etc/os-release:NAME="Debian GNU/Linux"
/etc/os-release:VERSION_ID="11"
/etc/os-release:VERSION="11 (bullseye)"
/etc/os-release:VERSION_CODENAME=bullseye
/etc/os-release:ID=debian

$ uname -a; grep -HvE "^#|URL" /etc/*release
Linux [servername] 5.10.63-v7l+ #1488 SMP Thu Nov 18 16:15:28 GMT 2021 armv7l GNU/Linux
/etc/os-release:PRETTY_NAME="Raspbian GNU/Linux 11 (bullseye)"
/etc/os-release:NAME="Raspbian GNU/Linux"
/etc/os-release:VERSION_ID="11"
/etc/os-release:VERSION="11 (bullseye)"
/etc/os-release:VERSION_CODENAME=bullseye
/etc/os-release:ID=raspbian
/etc/os-release:ID_LIKE=debian

$ uname -a; grep -HvE "^#|URL" /etc/*release
Linux [servername] 5.10.63-v7l+ #1488 SMP Thu Nov 18 16:15:28 GMT 2021 armv7l GNU/Linux
/etc/os-release:PRETTY_NAME="Raspbian GNU/Linux 11 (bullseye)"
/etc/os-release:NAME="Raspbian GNU/Linux"
/etc/os-release:VERSION_ID="11"
/etc/os-release:VERSION="11 (bullseye)"
/etc/os-release:VERSION_CODENAME=bullseye
/etc/os-release:ID=raspbian
/etc/os-release:ID_LIKE=debian
```

### Netdata build info

```shell
$ docker exec netdata netdata -W buildinfo
Version: netdata v1.33.0-9-nightly
Configure options:  '--prefix=/usr' '--sysconfdir=/etc' '--localstatedir=/var' '--libexecdir=/usr/libexec' '--libdir=/usr/lib' '--with-zlib' '--with-math' '--with-user=netdata' '--without-bundled-protobuf' '--with-bundled-libJudy' 'CFLAGS=' 'LDFLAGS='
Install type: unknown
Features:
    dbengine:                   YES
    Native HTTPS:               YES
    Netdata Cloud:              YES
    ACLK Next Generation:       YES
    ACLK-NG New Cloud Protocol: YES
    ACLK Legacy:                NO
    TLS Host Verification:      YES
    Machine Learning:           YES
    Stream Compression:         YES
Libraries:
    protobuf:                YES (system)
    jemalloc:                NO
    JSON-C:                  YES
    libcap:                  NO
    libcrypto:               YES
    libm:                    YES
    tcalloc:                 NO
    zlib:                    YES
Plugins:
    apps:                    YES
    cgroup Network Tracking: YES
    CUPS:                    NO
    EBPF:                    NO
    IPMI:                    YES
    NFACCT:                  NO
    perf:                    YES
    slabinfo:                YES
    Xen:                     NO
    Xen VBD Error Tracking:  NO
Exporters:
    AWS Kinesis:             NO
    GCP PubSub:              NO
    MongoDB:                 YES
    Prometheus Remote Write: YES
```

### Additional info

I have reviewed other issues raises and done a broader internet search with no solution.

**Comment by ilyam8** [comment]
Hi, @instantdreams. You need to mount the docker socket. Netdata uses it for docker containers name resolution.

> /var/run/docker.sock:/var/run/docker.sock:ro

**Comment by instantdreams** [comment]
This fixed the issue. Thank you very much!

**Comment by instantdreams** [comment]
On reading this, I feel there is still an issue that needs to be addressed.

The documentation section [Giving group access to the Docker socket](https://learn.netdata.cloud/docs/agent/packaging/docker/#giving-group-access-to-the-docker-socket-less-safe) explains that adding the PGID value into the environment section of the docker-compose will grant access but does not explicitly state that the docker socket also needs to be added.

It is explicitly mentioned in the other two methods. Can the documentation be adjusted perhaps?

**Comment by ilyam8** [comment]
@instantdreams you are right and that is a good suggestion. Fixed in #12079.
